### ISSUE TITLE
Ansible origin install fails with "package_found": false

### ISSUE THREAD

**Issue by go-faustino** [openshift/openshift-ansible#4276]
I'm trying to install openshift origin on a cluster, and the ansible playbook fails with the following error:

`TASK [openshift_version : Get available origin version] ***********************************************************************************************************************
fatal: [totlvdmoshd0001.totta.dev.corp]: FAILED! => {"changed": false, "failed": true, "msg": {"cmd": "/usr/bin/repoquery --plugins --quiet --pkgnarrow=repos --queryformat=%{version}|%{release}|%{arch}|%{repo}|%{version}-%{release} --config=/tmp/tmp5BOweZ origin", "package_found": false, "results": {}, "returncode": 1, "stderr": "Repository centos-openshift-origin is listed more than once in the configuration\nRepository centos-openshift-origin-testing is listed more than once in the configuration\nRepository centos-openshift-origin-debuginfo is listed more than once in the configuration\nRepository centos-openshift-origin-source is listed more than once in the configuration\nfailure: repodata/repomd.xml from centos-openshift-origin: [Errno 256] No more mirrors to try.\nhttp://mirror.centos.org/centos/7/paas/x86_64/openshift-origin/repodata/repomd.xml: [Errno 14] curl#6 - \"Could not resolve host: mirror.centos.org; Name or service not known\"\n", "stdout": ""}}
`

##### Version

* Ansible version is 2.3.0.0

##### Additional Information

* OS: RHEL 7.3
* The inventory file is as follows:

```
# Create an OSEv3 group that contains the master, nodes, etcd, and lb groups.
# The lb group lets Ansible configure HAProxy as the load balancing solution.
# Comment lb out if your load balancer is pre-configured.
[OSEv3:children]
masters
nodes
etcd


# host group for masters
[masters]
master1
master2 openshift_schedulable=true openshift_node_labels="{'region': 'infra', 'zone': 'default'}"


# host group for etcd
[etcd]
master1
master2

# host group for nodes, includes region info
[nodes]
node1 openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
node2 openshift_node_labels="{'region': 'primary', 'zone': 'west'}"

[OSEv3:vars]

openshift_release=v1.5.0

# SSH user, this user should allow ssh based auth without requiring a password
ansible_ssh_user=openshift

# If ansible_ssh_user is not root, ansible_become must be set to true
ansible_become=true

deployment_type=origin

openshift_master_identity_providers=[{'name': 'allow_all', 'login': 'true', 'challenge': 'true', 'kind': 'AllowAllPasswordIdentityProvider'}]

openshift_master_cluster_method= native
openshift_master_cluster_hostname= hostname1
openshift_master_cluster_public_hostname= hostname1

# enable ntp on masters to ensure proper failover
openshift_clock_enabled= true
```

**Comment by sdodson** [comment]
@go-faustino can you check your resolv.conf? We have a dispatcher script that alters /etc/resolv.conf but it shouldn't produce a configuration that's invalid. If you `chmod -x /etc/NetworkManager/dispatcher.d/99-origin-dns.sh` then restart NetworkManager can you resolve mirror.centos.org again?

**Comment by go-faustino** [comment]
@sdodson Hi, thanks for your reply! 

Resolv.conf, in my case, has a private DNS server, I'm behind a proxy. Following your clue, I included the proxy parameters in my hosts file and openshift-ansible/playbooks/byo/config.yml. But my particular problem was caused by the exclude clause added to yum.conf, the origin package wasn't listing using repoquery. Deleting the exclude was what solved it.

Now I'm further ahead, but getting a "no such file or directory" error on command "swapoff --all" on my nodes, trying to solve that now.
